liquibase-integration-tests/src/test/resources/liquibase/extension/testing/command/update.test.groovy

```diff
@@ -42,6 +42,8 @@ Optional Args:
     OBFUSCATED
   showSummary (UpdateSummaryEnum) Type of update results summary to show.  Values can be 'off', 'summary', or 'verbose'.
     Default: SUMMARY
+  showSummaryOutput (UpdateSummaryOutputEnum) Where to report the results summary. Values can be 'log', 'console', or 'all'.
```

Comment:
I would update this description to something like:
```
showSummaryOutput (UpdateSummaryOutputEnum) Summary output to report update summary results. Values can be 'log', 'console', or 'all'.
   Default: ALL
```

Reply:
Thank you, I've updated the description in all the place where it was used.